ui/app/src/Utils/Types.elm

```diff
@@ -1,20 +1,19 @@
 module Utils.Types exposing (..)
 
-import Http
 import Time
 
 
-type ApiResponse e a
+type ApiData a
     = Initial
     | Loading
-    | Failure e
+    | Failure String
```

Comment:
Having a string instead of a type variable forces us to drop information, doesn't it? Can't we keep the type variable here and convert an error to a string in the view functions?